packages/loaders/github/src/index.ts

```diff
@@ -30,12 +32,12 @@ export interface GithubLoaderOptions extends BaseLoaderOptions {
   /**
    * A GitHub access token
    */
-  token: string;
+  token?: string;
```

Comment:
When I was testing GitHub API, this token is required, otherwise we'd get a 401.

So we might have to keep this as required. 🙂

Reply:
Ok, I added `process.env.GITHUB_TOKEN` by default.